From c6590c289ed330292ff21e28630a108066642453 Thu, 26 Oct 2017 19:08:05 +0200
From: Andrew Branson <andrew.branson@jollamobile.com>
Date: Thu, 26 Oct 2017 19:07:31 +0200
Subject: [PATCH] Fix build by disabling gtk-doc generation and git calls

diff --git a/Makefile.am b/Makefile.am
index ce8e26f..117edb3 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -1,6 +1,4 @@
-DISTCHECK_CONFIGURE_FLAGS=--enable-gtk-doc
-
-COMMON_SUBDIRS = ext docs common tests
+COMMON_SUBDIRS = ext common tests
 SUBDIRS = $(LIBAV_SUBDIRS) $(COMMON_SUBDIRS) pkgconfig
 DIST_SUBDIRS = $(COMMON_SUBDIRS) gst-libs pkgconfig
 
@@ -44,9 +42,6 @@
 	$(top_builddir)/common/m4/ltversion.m4 \
 	$(top_builddir)/common/m4/lt~obsolete.m4
 
-CRUFT_DIRS = \
-	$(top_srcdir)/docs/plugins/tmpl
-
 include $(top_srcdir)/common/cruft.mak
 
 all-local: check-cruft
diff --git a/autogen.sh b/autogen.sh
index d1a2054..1bb25a6 100755
--- a/autogen.sh
+++ b/autogen.sh
@@ -17,12 +17,12 @@
 srcfile=gst-libav.doap
 
 # Make sure we have common
-if test ! -f common/gst-autogen.sh;
-then
-  echo "+ Setting up common submodule"
-  git submodule init
-fi
-git submodule update
+#if test ! -f common/gst-autogen.sh;
+#then
+#  echo "+ Setting up common submodule"
+#  git submodule init
+#fi
+#git submodule update
 
 # source helper functions
 if test ! -f common/gst-autogen.sh;
diff --git a/configure.ac b/configure.ac
index 8a3c82d..b757e47 100644
--- a/configure.ac
+++ b/configure.ac
@@ -96,10 +96,6 @@
 AC_PATH_PROG(VALGRIND_PATH, valgrind, no)
 AM_CONDITIONAL(HAVE_VALGRIND, test ! "x$VALGRIND_PATH" = "xno")
 
-dnl check for documentation tools
-GTK_DOC_CHECK([1.12])
-AG_GST_PLUGIN_DOCS([1.12])
-
 dnl *** checks for libraries ***
 
 dnl check for libm, for sin()
@@ -500,9 +496,6 @@
 ext/libswscale/Makefile
 gst-libs/Makefile
 gst-libs/ext/Makefile
-docs/Makefile
-docs/plugins/Makefile
-docs/version.entities
 tests/Makefile
 tests/check/Makefile
 tests/files/Makefile
