From d4d86efaa44b2464a77b45c7e33d4a1418bf47dc Mon Sep 17 00:00:00 2001
From: Andrew Branson <andrew.branson@jollamobile.com>
Date: Tue, 8 Oct 2019 18:20:32 +0200
Subject: [PATCH] Re-enable AAC encoder. JB#42353

---
 ext/libav/gstavaudenc.c | 13 +------------
 1 file changed, 1 insertion(+), 12 deletions(-)

diff --git a/ext/libav/gstavaudenc.c b/ext/libav/gstavaudenc.c
index 3ff6432..4f02e3f 100644
--- a/ext/libav/gstavaudenc.c
+++ b/ext/libav/gstavaudenc.c
@@ -768,7 +768,6 @@ gst_ffmpegaudenc_register (GstPlugin * plugin)
 
   while ((in_plugin = (AVCodec *) av_codec_iterate (&i))) {
     gchar *type_name;
-    guint rank;
 
     /* Skip non-AV codecs */
     if (in_plugin->type != AVMEDIA_TYPE_AUDIO)
@@ -833,17 +832,7 @@ gst_ffmpegaudenc_register (GstPlugin * plugin)
       }
     }
 
-    switch (in_plugin->id) {
-        /* avenc_aac: see https://bugzilla.gnome.org/show_bug.cgi?id=691617 */
-      case AV_CODEC_ID_AAC:
-        rank = GST_RANK_NONE;
-        break;
-      default:
-        rank = GST_RANK_SECONDARY;
-        break;
-    }
-
-    if (!gst_element_register (plugin, type_name, rank, type)) {
+    if (!gst_element_register (plugin, type_name, GST_RANK_SECONDARY, type)) {
       g_free (type_name);
       return FALSE;
     }
-- 
2.25.1

